packages/web-components/fast-foundation/src/progress-ring/progress-ring.template.ts

```diff
@@ -23,38 +28,21 @@ export function progressRingTemplate(
             ${when(
                 x => typeof x.value === "number",
                 html<FASTProgressRing>`
-                    <svg
-                        class="progress"
-                        part="progress"
-                        viewBox="0 0 16 16"
-                        slot="determinate"
+                    <span
+                        class="determinate"
+                        part="determinate"
+                        style="--percent-complete: ${x => x.percentComplete}"
                     >
-                        <circle
-                            class="background"
-                            part="background"
-                            cx="8px"
-                            cy="8px"
-                            r="7px"
-                        ></circle>
-                        <circle
-                            class="determinate"
-                            part="determinate"
-                            style="stroke-dasharray: ${x =>
-                                (progressSegments * x.percentComplete) /
-                                100}px ${progressSegments}px"
-                            cx="8px"
-                            cy="8px"
-                            r="7px"
-                        ></circle>
-                    </svg>
+                        ${x => options.determinateIndicator ?? defaultRingTemplate}
+                    </span>
                 `
             )}
             ${when(
                 x => typeof x.value !== "number",
                 html<FASTProgressRing>`
-                    <slot name="indeterminate">
-                        ${options.indeterminateIndicator || ""}
-                    </slot>
+                    <span class="indeterminate" part="indeterminate">
+                        ${x => options.indeterminateIndicator ?? defaultRingTemplate}
+                    </span>
```

Comment:
Commenting to track, this should remain a slot

Reply:
In both cases, we should remove the hard-link to the defaultRingTemplate. Let that be driven solely by the options. CLI can set up the code that provides the default via the generated options code. Then customers can change this without having to pay for the default template that can't be tree-shaken.